src/taxes/views/TaxClassesList.tsx

```diff
@@ -89,15 +104,23 @@ export const TaxClassesList: React.FC<TaxClassesListProps> = ({ id }) => {
     },
   });
 
-  const handleCreateTaxClass = async (input: TaxClassCreateInput) => {
+  const createTaxClass = async (
+    data: TaxClassesPageFormData,
+  ): Promise<CreateMetadataHandlerFunctionResult<
+    TaxClassCreateErrorFragment
+  >> => {
     const res = await taxClassCreateMutation({
       variables: {
-        input,
+        input: createTaxClassCreateInput(data),
       },
     });
-    refetch();
-    navigate(res?.data?.taxClassCreate?.taxClass?.id);
-    return res;
+
+    const taxClassCreate = res?.data?.taxClassCreate;
+
+    return {
+      id: taxClassCreate?.taxClass?.id,
+      errors: taxClassCreate?.errors,
+    };
```

Comment:
is it mean that both can be undefined/null?

Reply:
Yes, these are types from backend schema